scripts/source-maintenance.sh

```diff
@@ -35,6 +40,11 @@ while [ $# -ge 1 ]; do
         KeepGoingDirective=$1
         shift
         ;;
+    --with-astyle)
+		ASTYLE=$2
```

Comment:
Indentation consistency. surrounding code uses 8 spaces.

Reply:
Fixing, although the whole script is a bit of mix-n-match between tabs and 4-spaces indents